Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/impl/apmlog/LogFilterControllerWithAPMLog.java

```diff
@@ -91,19 +91,41 @@ private void onInvokeEvent(String payload) {
     private void onInvokeExtEvent(JSONObject param) {
         String type = (String) param.get("type");
         FilterType filterType = getFilterType(type);
+
+
+        Map<String, Object> payload = getEditorOptionPayload(filterType, param);
+        if (payload == null || payload.isEmpty()) {
+            return;
+        }
+
+        Clients.showBusy("Launch Filter Dialog ...");
+
+        LogFilterRule rule = getLastMatchedRuleWithValues(filterType, payload,
```

Comment:
When click on the "Activities" heading in the Log Stats box in PD, should it show only the "last matched rule" for "Activities" or should it show all the matched rules for "Activities". The latter seems more reasonable.

Reply:
Ok for now.